OJS.Workers.ExecutionStrategies/Java/JavaSpringAndHibernateProjectExecutionStrategy.cs

```diff
@@ -152,6 +99,12 @@ protected override string ClassPathArgument
             try
             {
                 submissionFilePath = this.CreateSubmissionFile(executionContext);
+                var isValid = this.ValidateFolderStructure(submissionFilePath);
+
+                if (!isValid)
+                {
+                    throw new ArgumentException("Folder structure is invalid!");
```

Comment:
Can you add more clear message to help the user understand what we are expecting to be a **valid** folder structure.